ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -182,72 +207,115 @@ const POPULATIONS = [
+    return acc
+  }, {})
+
+  const catalogs = Object.keys(popupPoputionByCat)
+  catalogs.sort()
+
   const freqContent = <div>{POPULATIONS.map(pop => <FreqSummary key={pop.field} variant={variant} {...pop} />)}</div>
 
-  const hasAcPops = POPULATIONS.filter(pop => populations[pop.field] && populations[pop.field].ac)
-  const hasGlobalAfPops = POPULATIONS.filter(pop => (
-    populations[pop.field] && populations[pop.field].filter_af &&
-    (populations[pop.field].filter_af !== populations[pop.field].af)))
   const hasHelpMessagePops = POPULATIONS.filter(
     pop => pop.helpMessage && populations[pop.field] && populations[pop.field].af !== null,
   )
-  const hasAcHetPops = POPULATIONS.filter(pop => populations[pop.field] && populations[pop.field].ac_het)
-  const hasMaxHlPops = POPULATIONS.filter(pop => populations[pop.field] && populations[pop.field].max_hl)
 
   return (
-    (hasAcPops.length || hasGlobalAfPops.length || hasHelpMessagePops.length || hasAcHetPops.length ||
-    hasMaxHlPops.length) ? (
+    (hasHelpMessagePops.length || catalogs.length) ? (
       <Popup position="top center" wide="very" trigger={freqContent}>
-        {hasGlobalAfPops.length > 0 && <Popup.Header content="Global AFs" />}
-        <Popup.Content>
-          {hasGlobalAfPops.map(pop => (
-            <div key={pop.field}>
-              {`${pop.fieldTitle}: ${populations[pop.field].af.toPrecision(pop.precision || 2)}`}
-            </div>
-          ))}
-        </Popup.Content>
-        {hasGlobalAfPops.length > 0 && hasAcPops.length > 0 && <VerticalSpacer height={5} />}
-        {hasAcPops.length > 0 && <Popup.Header content="Allele Counts" />}
-        <Popup.Content>
-          {hasAcPops.map(pop => (
-            <div key={pop.field}>
-              {`${pop.fieldTitle}: ${populations[pop.field].ac} out of ${populations[pop.field].an}`}
-            </div>
-          ))}
-        </Popup.Content>
-        {((hasGlobalAfPops.length > 0 || hasAcPops.length > 0) &&
-            (hasAcHetPops.length > 0 || hasMaxHlPops.length > 0)) && <VerticalSpacer height={5} />}
-        {(hasAcHetPops.length > 0 || hasMaxHlPops.length > 0) && <Popup.Header content="Heteroplasmy" />}
-        <Popup.Content>
-          {hasAcHetPops.map(pop => (
-            <div key={pop.field}>
-              {`${pop.fieldTitle} heteroplasmic AF: ${populations[pop.field].af_het?.toPrecision(pop.precision || 2)}
-              AC_het=${populations[pop.field].ac_het} out of ${populations[pop.field].an}`}
-            </div>
-          ))}
-          {hasMaxHlPops.map(pop => (
-            <div key={pop.field}>
-              {`${pop.fieldTitle} max observed heteroplasmy:
-              ${populations[pop.field].max_hl.toPrecision(pop.precision || 2)}`}
-            </div>
-          ))}
-        </Popup.Content>
+        {catalogs.map((catalog, i) => (
+          <div key={catalog}>
+            {(i > 0) && <VerticalSpacer height={5} />}
+            <Popup.Header content={catalog} />
```

Comment:
The `Popup.Header` component's style is changed as below picture.
<img src="https://user-images.githubusercontent.com/33550059/187272214-587eb28f-cdf5-41e3-bbc3-87a878156b9d.png" width=40% hight=40% />

Reply:
The issue here is you are putting this all inside a `div` which is not the correct nesting for the styling to work. The element grouping should be unnecessary, you should be able to change this to render a top-level list by using a reduce instead of a map:
```
{catalogs.reduce((acc, catalog, i) => ([
  ...acc, 
  i > 0 ? <VerticalSpacer key={`spacer-${category}`} height={5} /> : null,
  <Popup.Header key={catalog} content={catalog} />,
  ...popupPoputionByCat[catalog].map((pop) => ...),
  ...etc
]), []}
```

Also, as a high level thing, "catalog" is a really confusing word in this context. I am really unsure what you meant for it to mean here, but I think "detailSection" would be a much better word, and for the list you could use the word "sections"

Reply:
Thank you. Update pushed.